src/components/core/createPost/index.tsx

```diff
@@ -172,11 +171,7 @@ const CreatePost = () => {
 
   return (
     <>
-      <motion.section
-        initial={{ opacity: 0, y: -350 }}
-        whileInView={{ opacity: 1, y: 0 }}
-        viewport={{ once: true }}
-        transition={{ duration: 0.1, type: "spring", stiffness: 110 }}
+      <div
```

Comment:
instead of remove, comment the functionality, we might need this later

Reply:
Yeah I would gladly add if needed I know framer-motion but the problem is sometimes that component renders and sometimes not and due to this I removed that animation to make stable user experience

Reply:
That is why I said, we might need later when we finish making the components

Reply:
added the changes